packages/faustwp-cli/utils/promptUserForTelemetryPref.ts

```diff
@@ -1,37 +0,0 @@
-import Configstore from 'configstore';
-import prompt from 'prompt';
-import { v4 as uuid } from 'uuid';
-
-prompt.start();
-
-/**
- * Prompt the user for their telemetry preferences.
- *
- * @param isInit boolean Based on this value, we will either save a new telemetry config, or update the "enabled" value in the config.
- * @param config The Configstore config
- */
-export const promptUserForTelemetryPref = async (
-  isInit: boolean,
-  config: Configstore,
-) => {
-  const { isTelemetryEnabled } = await prompt.get({
-    properties: {
-      isTelemetryEnabled: {
-        description:
-          'Would you like to enable telemetry in Faust? This helps us make more informed feature decisions. true/false',
-        required: true,
-        type: 'boolean',
-      },
-    },
-  });
-
-  if (isInit) {
-    config.set('telemetry', {
-      notifiedAt: new Date().getTime(),
-      anonymousId: uuid(),
-      enabled: isTelemetryEnabled,
-    });
-  } else {
-    config.set('telemetry.enabled', isTelemetryEnabled);
-  }
```

Comment:
This functionality was removed but is important.

We are no longer determining if it's the user first time setting preferences. We need this to happen so we can assign them a unique id, set the notified time, etc.

Since we are no longer prompting, and we don't have a clear delineation if we have notified the user yet, maybe we do the following:
1. Check if the user has preferences
2. If not, show them a message saying Faust has telemetry, and create their preferences with `enabled: false`.
3. Now that we have their preferences and when they were notified, we can stop showing them the above mentioned console log